PhysicsTools/PatUtils/python/L1ECALPrefiringWeightProducer_cfi.py

```diff
@@ -0,0 +1,11 @@
+import FWCore.ParameterSet.Config as cms
+
+prefiringweight = cms.EDProducer("L1ECALPrefiringWeightProducer",
```

Comment:
@lathomas @perrotta this line seems inconsistent with

descriptions.add("l1ECALPrefiringWeightProducer",desc);

as it should be 

descriptions.add("prefiringweight", desc); 

what is the reason for this discrepancy? And in general to have both the fillDescriptions and the external cfi?